token/cli/src/main.rs

```diff
@@ -4371,4 +4619,38 @@ mod tests {
         let account = config.rpc_client.get_account(&token_pubkey).await;
         assert!(account.is_err());
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn enable_required_transfer_memos() {
+        let (test_validator, payer) = new_validator_for_test().await;
+        let program_id = spl_token_2022::id();
+        let config = test_config(&test_validator, &payer, &program_id);
+        let token = create_token(&config, &payer).await;
+        let token_account = create_associated_account(&config, &payer, token).await;
+        // Enabling required transfer memos using mint account as argument
+        let result = process_test_command(
+            &config,
+            &payer,
+            &[
+                "spl-token",
+                CommandName::EnableRequiredTransferMemos.into(),
+                &token.to_string(),
+            ],
+        )
+        .await;
+        result.unwrap();
```

Comment:
i would fetch the account and confirm the extension exists in its types here, and that it doesnt after the disable call

Reply:
resolved with d37c851